_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,350 @@
+  to be used for later versions of segwit witness programs but
+  a [length extension mutation][news78 bech32 mutability] was discovered
+  that necessitates using a modified bech32 address format, possibly
+  called "bech32m", for Pay-to-Taproot (P2TR) segwit v1 witness programs.
+
+- [What makes cross input signature aggregation complicated to implement?]({{bse}}100216)
+  Michael Folkson references a Bitcoin dev mailing list [post by AJ Towns][aj signature aggregation]
+  in explaining challenges to implementing cross input signature aggregation in Bitcoin.
+
+- [How do BIP8 and BIP9 differ, how are they alike?]({{bse}}100490)
+  Murch provides an overview of early soft fork activation methods using the
+  block version, BIP9's multiple-proposal-supporting activation method, and BIP8's block
+  height and lock-in adjustments to BIP9.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.0rc3][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.  Note, the rc3 version of the
+  [signed binary][macos signed] will not run due to a problem with the
+  code signing tool.  The [unsigned version][macos unsigned] (which can
+  still be [verified][pgp verification] with PGP) should run if opened
+  using the right-click (or control-click) context menu.
+
+- Bitcoin Core 0.20.2rc1 and 0.19.2rc1 are expected to be
+  [available][bitcoincore.org/bin] sometime after the publication of
+  this newsletter.  They contain several bug fixes, such as an
+  improvement described in [Newsletter #110][news110 bcc19620] that will
+  prevent them from redownloading future taproot transactions that they
+  don't understand.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20564][] makes two changes to the way that Bitcoin Core
+  signals support for `addrv2` messages ([BIP155][]):
+
+  - *Protocol version:* Bitcoin Core will only negotiate support for `addrv2` messages with peers that signal
+    they are using P2P version 70016 or higher. This restriction isn't required by BIP155,
+    but release testing has revealed that some other implementations will disconnect
+    from Bitcoin Core if they receive any unknown message, including
+    `sendaddrv2`. This change may be reverted in future versions of Bitcoin Core,
+    so other implementations are advised to tolerate unknown P2P messages at any time in the connection.
```

Comment:
I think 'at any time in' is ok. We don't want to drop this part entirely because it emphasizes that unknown messages are allowed before and after the verack.